app/javascript/lib/honeycrisp.js

```diff
@@ -216,7 +225,15 @@ var noneOfTheAbove = (function() {
             $noneCheckbox.click(function(e) {
                 $otherCheckboxes.prop('checked', false);
                 $otherCheckboxes.parent().removeClass('is-selected');
+
+                // If we just unchecked an <input> with a follow-up, let's reset the follow-up questions
+                // so it hides properly.
+                var $enclosingFollowUp = $noneCheckbox.closest('.question-with-follow-up');
+                if ($enclosingFollowUp) {
+                    followUpQuestion.update($enclosingFollowUp);
+                }
```

Comment:
also part of https://github.com/codeforamerica/honeycrisp-gem/commit/c05fb0b74c1ecf67a62d9735c5dc77240376beb2